appsec-kit-v24/pom.xml

```diff
@@ -32,6 +32,21 @@
             <version>${vaadin24.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin</artifactId>
+            <version>${vaadin24.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>flow-html-components</artifactId>
+            <version>24.1.10</version>
```

Comment:
When we have the Flow property, let's use it here.

Reply:
Changed to use Flow property.